cardano-node-chairman/src/Testnet/Byron.hs

```diff
@@ -112,7 +120,26 @@ testnet H.Conf {..} = do
     H.createDirectoryIfMissing dbDir
     H.createDirectoryIfMissing $ tempBaseAbsPath </> "" <> socketDir
 
-    H.copyFile (baseConfig </> "topology-node-" <> si <> ".json") (tempAbsPath </> "topology-node-" <> si <> ".json")
+    let otherPorts = let (as, bs) = L.splitAt i allPorts in as <> L.drop 1 bs
+
+    H.lbsWriteFile (tempAbsPath </> "topology-node-" <> si <> ".json") $ J.encode $
+      J.object
```

Comment:
Probably should pull that out as a separate top level or `where` clause value rather than defining it inline.

Reply:
Move to top-level `dropNth` function.